configs/sst_cs_databases-unwanted.yaml

```diff
@@ -1,12 +1,10 @@
 document: feedback-pipeline-unwanted
 version: 1
 data:
-  name: Unwanted packages from CS Apps SST
-  description: Packages we do not want to ship for CS Apps SST
-  maintainer: sst_cs_apps
+  name: Unwanted packages from CS Databases SST
+  description: Packages we do not want to ship for CS Databases SST
+  maintainer: sst_cs_databases
   unwanted_packages:
-    - js-uglify
-    - web-assets-devel
     - mercurial # Hard to maintain while not the most popular choice anymore.
```

Comment:
I guess this should be removed as well now that it is covered in [sst_cs_base_utils-unwanted.yaml](https://github.com/minimization/content-resolver-input/pull/1321/files#diff-0decb157f168bbd5a0470ef8635a676417526e1be45d4a38ac03e68b4b31ae28)?

Reply:
yes, will fix